src/devices/ExplorerHat/GpioControllerExtensions.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Device.Gpio;
+
+namespace Iot.Device.ExplorerHat
+{
+    /// <summary>
+    /// Extension methods for <see cref="GpioController"/>
+    /// </summary>
+    public static class GpioControllerExtensions
```

Comment:
seems this is not doing much, should probably be removed 😄

Reply:
Sorry